res/css/views/right_panel/_TimelineCard.pcss

```diff
@@ -170,16 +170,12 @@ limitations under the License.
         flex-basis: 48px; /* 12 (padding on message list) + 36 (padding on event lines) */
     }
 
-    .mx_GenericEventListSummary_unstyledList, /* RR next to a message on the event list summary */
+    .mx_GenericEventListSummary_unstyledList,
     .mx_RoomView_MessageList {
-        /* RR next to a message on the messsge list */
         .mx_EventTile[data-layout="bubble"] {
+            /* RR next to a message on the event list summary and the messsge list */
             .mx_ReadReceiptGroup {
-                /* 6px: scroll bar width (magic number) */
```

Comment:
It looks that the magic number should in fact have been `var(--BaseCard_padding-inline)`.